src/packages/@ncigdc/modern_components/AffectedCasesTable/AffectedCasesTable.js

```diff
@@ -362,11 +362,13 @@ const Component = compose(
                     gender: c.demographic ? c.demographic.gender : '',
                     age_at_diagnosis: ageDisplay(diagnosis.age_at_diagnosis),
                     tumor_stage: diagnosis.tumor_stage,
-                    days_to_last_follow_up: (diagnosis.days_to_last_follow_up ||
-                      0)
-                      .toLocaleString(),
-                    days_to_death: (diagnosis.days_to_death || 0)
-                      .toLocaleString(),
+                    days_to_last_follow_up: (diagnosis.days_to_last_follow_up !=
```

Comment:
yea I could see `== null` getting change to `=== null` by a linter or someone looking at the code later.